proj.py

```diff
@@ -0,0 +1,134 @@
+import argparse
+import matplotlib.pyplot as plt
+import numpy as np
+import os
+import pandas as pd
+import seaborn as sns
+import sqlite3
+
+
+CON = ""
+C = ""
+DF = ""
+
+
+def load_data():
+    return pd.read_csv(r'C:\Users\Dell\OneDrive\Desktop\netflix_titles.csv')
+
+
+def db_connection():
+    return sqlite3.connect("netflix.db")
+
+
+def init_app():
+    global DF
+    DF = load_data()
+    global CON
+    CON = db_connection()
+    global C
+    C = CON.cursor()
+    DF.to_sql('MyTable', CON, if_exists='append', index=False)
+    DF = pd.read_sql_query("SELECT * from MyTable", CON)
+    DF = DF.drop_duplicates(['title', 'country', 'type', 'release_year'])
+    DF['date_added'] = pd.to_datetime(DF['date_added'])
+    for ratings in DF.index:
+        if DF.loc[ratings, 'rating'] == 'UR':
+            DF.loc[ratings, 'rating'] = 'NR'
```

Comment:
because the data has two different rating names for the same rating i.e. Un Rated = UR and Not Rated = NR and the standard rating is NR. It has no functional advantage though.